sdk/tests/utests/Job_tests.cpp

```diff
@@ -0,0 +1,279 @@
+/**
+ * Copyright (c) 2022 Robert Bosch GmbH
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <condition_variable>
+#include <mutex>
+#include <thread>
+
+#include <gtest/gtest.h>
+
+#include "sdk/Job.h"
+
+using namespace velocitas;
+using namespace std::chrono_literals;
+
+TEST(Test_Job, callCtor_none_noFailure) {
```

Comment:
Prefer positive case description

```suggestion
TEST(Test_Job, callCtor_emptyLambda_success) {
```